index.js

```diff
@@ -1,32 +1,9 @@
 /*jshint node:true */
-
-var request = require('request')
+var charted = require('./lib/charted.js')
 var express = require('express')
 var app = express()
 
-app.get('/get', function (req, res) {
-  if (!req.query.url) {
-    res.status(400).send('Bad Request: no url provided')
-    return
-  }
-
-  request(decodeURIComponent(req.query.url), function (err, resp, body) {
-    if (err) {
-      res.status(400).send('Bad Request: ' + err)
-      return
-    }
-
-    if (resp.statusCode != 200) {
-      res.status(400).send('Bad Request: response status code was not 200')
-      return
-    }
-
-    res.set('Content-Type', 'text/plain')
-    res.status(200).send(body)
-  })
-})
-
-app.use(express.static('pub'))
+charted(app, '')
```

Comment:
The second parameter here, an empty string, is very non-obvious. Maybe change line 11 in charted.js to be `charted_root + 'get'` so that you could pass a `/` as a second parameter? Or even make API something like this?

```javascript
charted(app)
  .withRoot('/')
```

This way it will be way easier to add new configuration flags to the library in future.